packages/common-all/src/store/NoteStore.ts

```diff
@@ -64,7 +54,11 @@ export class NoteStore implements Disposable, INoteStore<string> {
         data: { ...metadata.data, body: "" },
       };
     }
-    const uri = NoteUtils.getURI({ note: metadata.data, wsRoot: this._wsRoot });
+    const uri = Utils.joinPath(
+      this._wsRoot,
+      VaultUtils.getRelPath(metadata.data.vault),
```

Comment:
`getURI` would check if the vault path is an absolute path vs a relative path. looks like this doesn't do that? also, should we extract this into its own function

Reply:
In what situations do vaults have absolute paths specified versus relative paths?